ruby/src/main/java/com/google/protobuf/jruby/RubyProtobuf.java

```diff
@@ -65,4 +66,17 @@ public static IRubyObject deepCopy(ThreadContext context, IRubyObject self, IRub
             return ((RubyMap) message).deepCopy(context);
         }
     }
+
+    /*
+     * call-seq:
+     *     Google::Protobuf.discard_unknown(msg)
+     *
+     * Discard unknown fields in the given message object and recursively discard
+     * unknown fields in submessages.
+     */
+    @JRubyMethod(name = "discard_unknown", meta = true)
+    public static IRubyObject discardUnkown(ThreadContext context, IRubyObject self, IRubyObject message) {
```

Comment:
Typo: discardUnkown -> discardUnknown

Reply:
Good catch. I'll push up a fix for this once the tests all run in case there are any other changes I need to make.

Reply:
All set